web-registry/src/components/organisms/SellOrdersTable/SellOrdersTable.Row.tsx

```diff
@@ -40,7 +40,7 @@ const getSellOrdersTableRow = ({
   },
 }: Props): React.ReactNode[] => [
   <Box sx={{ color: 'info.main' }}>{id}</Box>,
-  <WithLoader isLoading={project?.name === undefined} variant="skeleton">
+  <WithLoader isLoading={!project?.name} variant="skeleton">
```

Comment:
Actually, the check on undefined was meant to prevent infinite loading in case the request failed.
See this file for more detail: https://github.com/regen-network/regen-web/blob/master/web-registry/src/pages/Marketplace/Storefront/Storefront.normalizer.ts#L108-L117
We should not keep the skeleton if the request has been finished.

Reply:
You can close this PR I think @victordelval , I made a PR here for it: https://github.com/regen-network/regen-web/pull/1377 that fixes this issue and keep handling the case I mentioned above.

Reply:
I see, perfect, thanks!